migrations/Version202410311205211101_taoQtiItem.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\oatbox\event\EventManager;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoItems\model\event\ItemCreatedEvent;
+use oat\taoItems\model\event\ItemDuplicatedEvent;
+use oat\taoQtiItem\model\event\ItemImported;
+use oat\taoQtiItem\model\UniqueId\Listener\ItemCreatedEventListener;
+
+/**
+ * phpcs:disable Squiz.Classes.ValidClassName
+ */
+final class Version202410311205211101_taoQtiItem extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        /** @var EventManager $eventManager */
+        $eventManager = $this->getServiceManager()->get(EventManager::SERVICE_ID);
+
+        $eventManager->attach(
```

Comment:
- [ ] @shpran Notice that on `ResourceCreated` event we also deal with that and there is already some listener created for this. We need to make sure we do not have discrepant logic among those event / listeners.

Reply:
I'm not triggering ResourceCreated event

Reply:
I think it is triggered by clone functionality

Reply:
Yes, but as I mentioned, I did not check for ResourceCreated event here 🙂 
I created new event for this - `InstanceCopiedEvent`, so now I have a specific event for specific action